src/wiki/development/build-instructions-macos.md

```diff
@@ -0,0 +1,154 @@
+---
+eleventyNavigation:
+  key: MacOS Build instructions
+  parent: Development
+  order: 4
+---
+# Build Instructions
+
+## Contents
+
+- [Getting the source](#getting-the-source)
+- [Building](#building)
+- [IDEs and Tooling](#ides-and-tooling)
+
+## Getting the source
+
+Clone the source code using git, and grab all the submodules:
+
+```bash
+git clone --recursive https://github.com/PrismLauncher/PrismLauncher.git
+cd PrismLauncher
+```
+
+**The rest of the documentation assumes you have already cloned the repository.**
+
+## Building
+
+### Install prerequisites
+
+- Install XCode Command Line tools.
+- Install the official build of CMake (<https://cmake.org/download/>).
+- Install extra-cmake-modules
+- Install JDK 8 (<https://adoptium.net/temurin/releases/?variant=openjdk8&jvmVariant=hotspot>).
+- Install Qt 5.12 or newer or any version of Qt 6 (recommended)
+
+Using [homebrew](https://brew.sh) you can install these dependencies with a single command:
+
+```bash
+brew update # in the case your repositories weren't updated
+brew install qt openjdk@17 cmake ninja extra-cmake-modules # use qt@5 if you want to install qt5
+```
+
+### XCode Command Line tools
+
+If you don't have XCode Command Line tools installed, you can install them with this command:
+
+```bash
+xcode-select --install
+```
+
+### Build
+
+Choose an installation path.
+
+This is where the final `PrismLauncher.app` will be constructed when you run `make install`. Supply it as the `CMAKE_INSTALL_PREFIX` argument during CMake configuration. By default, it's in the dist folder, under PrismLauncher.
+
+```bash
+mkdir build
+cd build
+cmake \
+ -DCMAKE_BUILD_TYPE=Release \
+ -DCMAKE_INSTALL_PREFIX:PATH="$(dirname $PWD)/dist/" \
+ -DCMAKE_PREFIX_PATH="/path/to/Qt/" \
+ -DQt5_DIR="/path/to/Qt/" \
+ -DCMAKE_OSX_DEPLOYMENT_TARGET=10.14 \
+ -DLauncher_QT_VERSION_MAJOR=6 \ # if you want to use Qt 6
+ -DENABLE_LTO=ON \ # if you want to enable LTO/IPO
+ -DLauncher_BUILD_PLATFORM=macOS
+#-DCMAKE_OSX_ARCHITECTURES="x86_64;arm64" # to build a universal binary (not recommended for development)
+ ..
+make install
+```
+
+Remember to replace `/path/to/Qt/` with the actual path. For newer Qt installations, it is often in your home directory. For the Homebrew installation, it's likely to be in `/opt/homebrew/opt/qt`.
+
+**Note:** The final app bundle may not run due to code signing issues, which
+need to be fixed with `codesign -fs -`.
+
+## IDEs and Tooling
```

Comment:
uh this part isn't only for macos

Reply:
nvm ignore this

Reply:
I just moved it how it was structured in the original single file, for which platforms would that apply?

Reply:
yeah ignore it, didn't notice :/